src/views/index.html

```diff
@@ -30,7 +30,16 @@ <h1>
     <p>Wildcards can be used in specified fields. Use _ for single characters or % for multiple characters.</p>
     <p><i>E.g. ("Miss_ng" will find "Missing", "Miss%" will find "Missing")</i></p>
     {% else %}
-    <p><i><u>Note:</u> Strict searching must be checked for searches using non-fuzzy fields (e.g. Badge)</i></p>
+    <div class="field-note">
+        <p><i><u>Note:</u> Strict searching must be checked for these fields to use non-fuzzy matching:</i></p>
```

Comment:
I think this might be more informative the other way around - instead of listing the fuzzy fields, this might be better as "Strict searching must be checked for the following fields to be used effectively", and then the list would be `if not entity.fuzzy` entities.